contact.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - About</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+
+<div class="container-contact-page"></div>
+  
+  <nav class="home-navigation">
+    <a href="index.html"><img class="lambda-black" src="img/lambda-black.png" alt="Lambda logo"></a>
+    <a href="index.html">Home</a>
+    <a href="about.html">About</a>
+    <a href="#">Products</a>
+    <a href="#">Blog</a>
+    <a href="contact.html">Contact</a>
+  </nav>
+
+  <section class="contact-page-top-content">
+    <h2>Hi</h2>
+      <p>Thank you for visiting our page and expressing interest!</p>
+      <p>If you would like to get in contact please send us an email using our contact form below.</p>
+  </section>  
+
```

Comment:
you could have looked up how to do a contact form or at less added a fake email address